features/bevragen/protocollering/protocollering.feature

```diff
@@ -131,17 +131,18 @@ Functionaliteit: Als burger wil ik zien wie welke gegegevens van mij heeft gezie
       | <gevraagde rubrieken>       |
 
       Voorbeelden:
-      | groep             | gevraagde rubrieken                    |
-      | geboorte          | 010310, 010320, 010330                 |
-      | nationaliteiten   | 040510, 046310, 046510, 048510         |
-      | overlijden        | 060810, 060820, 060830                 |
-      | europeesKiesrecht | 133110, 133130                         |
+      | groep                                | gevraagde rubrieken            |
+      | geboorte                             | 010310, 010320, 010330         |
+      | nationaliteiten                      | 040510, 046310, 046510, 048510 |
```

Comment:
Deze protocollering verschilt met wat in [features/bevragen/persoon/nationaliteit/protocollering/protocollering-gevraagde-rubrieken.feature](https://github.com/BRP-API/Haal-Centraal-BRP-bevragen/pull/1758/files#diff-8f8bbd647539f53f16a2f2497b86ab4c5abf3bb98bf23a5375126aeccf1310a0) regel 39 staat. Welke moet worden gehanteerd?

Reply:
is geen handig voorbeeld en heb ik verwijderd. [features/bevragen/persoon/nationaliteit/protocollering/protocollering-gevraagde-rubrieken.feature](https://github.com/BRP-API/Haal-Centraal-BRP-bevragen/pull/1758/files#diff-8f8bbd647539f53f16a2f2497b86ab4c5abf3bb98bf23a5375126aeccf1310a0) regel 39 is de juiste